content/vendor/guides/helm-chart.md

```diff
@@ -107,27 +108,28 @@ spec:
 EOF
 ```
 
+You may have noticed that we have also used this opportunity to set just two of the values for this chart: `adminUser` and `adminPassword`. We'll hardcode them to `admin`/`admin` for now, but we'll make these user-configurable later.
+
 * * *
 
 ## Releasing and Testing
 
 Make a new release with the Helm chart in your kots app.
 
 ```shell
-$ cd ~/helm-grafana
-
-$ make release
+cd ~/helm-grafana
+make release
```

Comment:
since this is a multi-line command block, I've removed the `$` signs so it can be easily copy-pasted, per our style guide: 

https://github.com/replicatedhq/help-center/blob/master/CONTRIBUTING.md#multi-line-code-snippets-should-not-have-the-bash-prompt